RyftUI/Source/Component/RyftThreeDsActionComponent.swift

```diff
@@ -33,10 +33,14 @@ public final class DefaultRyftThreeDsActionHandler: RyftThreeDsActionHandler {
             scheme: action.scheme
         )
         DispatchQueue.main.async {
-            self.threeDsSdk.authenticate(authenticationParameters: params) { maybeError in
-                completion(maybeError)
-                self.threeDsSdk.cleanup()
```

Comment:
Note I can no longer find this referenced/suggested in the documentation so I've removed it.
In comparison to our Android SDK we don't do any explicit 3DS SDK clean up from what I understand so this matches up.